app/components/data_table/row_component.rb

```diff
@@ -0,0 +1,26 @@
+module DataTable
+  class RowComponent < GovukComponent::Base
+    renders_many :cells, lambda { |colname: nil, colspan: 1, scope: 'col', classes: [], numeric: false, text: nil|
```

Comment:
Wondering if these params to the components could be wrapped in some object - they are passed in a lot in they various classes. Perhaps `CellComponentConfig`?

Reply:
They're subtly different. The lamda args set defaults for a column header. (Which reminds me, I should rename the HeaderRowComponent.)